packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -168,7 +168,9 @@ export function createAutocompleteDom<TItem extends BaseItem>({
     const detachedCancelButton = createDomElement('button', {
       class: classNames.detachedCancelButton,
       textContent: translations.detachedCancelButtonText,
-      onClick() {
+      onClick(event: MouseEvent) {
+        event.preventDefault();
```

Comment:
this line isn't needed if we pass `type: 'button'`, right?

Reply:
You're right! I made the change in bc6611e2. I set it to the detached search button as well.